packages/nextra/package.json

```diff
@@ -111,6 +111,7 @@
     "@mdx-js/mdx": "^2.3.0",
     "@mdx-js/react": "^2.3.0",
     "@napi-rs/simple-git": "^0.1.8",
+    "clsx": "^1.2.1",
```

Comment:
Instead of using `clsx` we can use something like `TypeToClassName` like we do with emojis, what do you think?

Reply:
I think is totally fine to use `clsx` in nextra, because I want to move other components from `nextra-theme-docs` to nextra too - Tabs, Steps, FileTree

Reply:
Sounds good! You can work on it, thank you!